hazelcast/src/main/java/com/hazelcast/internal/management/request/WanCheckConsistencyRequest.java

```diff
@@ -51,12 +54,19 @@ public int getType() {
     @Override
     public void writeResponse(ManagementCenterService mcs, JsonObject out) throws Exception {
         WanCheckConsistencyOperation operation = new WanCheckConsistencyOperation(schemeName, publisherName, mapName);
-        Object operationResult = resolveFuture(mcs.callOnThis(operation));
+        InternalCompletableFuture<Object> future = mcs.callOnThis(operation);
         JsonObject result = new JsonObject();
-        if (operationResult == null) {
-            result.add("result", SUCCESS);
-        } else {
-            result.add("result", operationResult.toString());
+        Object operationResult;
+        try {
+            operationResult = future.get();
+            if (operationResult == null) {
+                result.add("result", SUCCESS);
+            } else {
+                result.add("result", operationResult.toString());
+            }
+        } catch (ExecutionException e) {
```

Comment:
Is it a good idea to include this logic into `ManagementCenterService.resolveFuture`, so it would return a `JsonObject` with the result? I can see that it's used in other requests, like `ChangeWanStateRequest` or `ClearWanQueuesRequest`.

Reply:
Nice suggestion! Done and removed some duplication. Also added stacktrace reporting to `ChangeWanStateRequest` and `ClearWanQueuesRequest`.